notebooks/scripts/plot_effect_estimation.py

```
@@ -98,4 +174,57 @@
 
 ax.plot([-0.25, 0.4], [-0.25, 0.4], color="#696969", linestyle="--", label="True ATE")
 ax.legend(ncol=4, frameon=False, bbox_to_anchor=(0.5, 1.1), loc="upper center")
+fig.savefig("figures/effect_estimation_different_effect_strength_a.png")
+fig
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)